aperturedb/ConnectionLoader.py

```diff
@@ -7,19 +7,28 @@
 
 
 class ConnectionGeneratorCSV(CSVParser.CSVParser):
-    """**ApertureDB Connection Data loader.**
+    """**ApertureDB Connection Data generator.**
 
     .. note::
-        Expects a csv file with the following columns:
+        Is backed by a csv file with the following columns:
 
             ``ConnectionClass``, ``Class1@PROP_NAME``, ``Class2@PROP_NAME`` ... ``PROP_NAME_N``, ``constraint_PROP1``
 
     Example csv file::
 
-        ConnectionClass,_Image@id,Person@id,confidence,id,constaint_id
-        has_image,321423532,42342522,0.4354,5432543254,5432543254
-        has_image,321423532,53241521,0.6432,98476542,98476542
+        ConnectionClass,_Image@id,_Descriptor@UUID,confidence,id,constaint_id
+        has_image,321423532,AID-0X3E,0.4354,5432543254,5432543254
+        has_image,42342522,BXY-AB1Z,0.6432,98476542,98476542
         ...
+
+    **ConnectionClass**: Arbitrary class name for the entity this would be saved as.
+
+    **<ClassName>@<PropertyName>** This is a special combination of Class Name and Property Name that can uniquely identify an entity.
```

Comment:
```suggestion
    **<ClassName>@<PropertyName>** This is a special combination of Class Name and 
    Property Name that can uniquely identify an entity.
```

Doesn't PEP8 check for line width?

Reply:
I don't think it runs on docstring (and thats good, or sphinx will get ruined)